src/PointwiseFunctions/AnalyticData/GhScalarTensor/Factory.hpp

```diff
@@ -0,0 +1,17 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "PointwiseFunctions/AnalyticData/ScalarTensor/KerrSphericalHarmonic.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace gh::AnalyticData {
```

Comment:
I would flip the namespaces, `gh::ScalarTensor::AnalyticData`. Probably need:
```cpp
/// ScalarTensor solutions wrapped for GH
namespace gh::ScalarTensor::AnalyticData {
``` 
I'm not sure why GRMHD is done this way, that needs to get fixed but not your fault :) (looks like it's my fault, actually :( sigh...)